config/vufind/config.ini

```diff
@@ -1763,10 +1763,27 @@ hide_holdings[] = "World Wide Web"
 ;
 ; Available options:
 ;    Channels - Display links to channels of content related to record
+;    Bookplate - Display a bookplate or something similar
 ;    Similar - Similarity based on Solr lookup
 ;    WorldCatSimilar - Similarity based on WorldCat lookup
 related[] = "Similar"
 
+; URLs for bookplates if enabled in the related config above. %s represents an
+; image name or donor code that can be used to identify an image. This will
+; come from a Solr field (donor_code by default).
+; bookplate_full = https://your-institution.edu/bookplates/%s-full.jpg
```

Comment:
Minor style suggestion: VuFind's convention in configuration files is that we put a space after the semi-colon for textual comments, but we do not put a space after the semi-colon for commented-out example settings. This makes it a little easier to visually distinguish between the two. To follow that convention, I'd recommend removing a few spaces from this file.

Reply:
Fixed.